reviews/2020-strimzi-annual.md

```diff
@@ -0,0 +1,73 @@
+# Strimzi Sandbox Annual Review
+
+Strimzi is a project focused on running Apache Kafka on Kubernetes. Apache Kafka is a leading platform for building real-time data pipelines. It provides support for high-throughput / low-latency messaging, as well as sophisticated development options that cover all the stages of a distributed data streaming pipeline, from ingestion, through storage to processing. But running it on Kubernetes can be complex and tedious. Strimzi is making it easy by using the operator pattern and declarative deployments. It provides operators not just for deploying Kafka clusters, but also for managing topics, users or connectors to make sure running Apache Kafka feels Kube-native.
+
+Strimzi also integrates with many other CNCF projects such as:
+* Kubernetes
+* Prometheus for metrics and monitoring
+* Open Policy Agent for authorization
+* KEDA for autoscaling of Kafka clients
+* Jaeger and Open Tracing for tracing
+
+Strimzi can be also installed using Helm charts or using OperatorHub.io which is part of the Operator Framework project.
+
+Strimzi joined CNCF Sandbox in September 2019.
```

Comment:
Could you include the link to the original presentation, to the annual review? It would help with setting the initial context for the reviewers.

Reply:
Thanks for your comment. I added the link.